librosa/core/audio.py

```diff
@@ -537,11 +537,15 @@ def resample(
         'polyphase'
             `scipy.signal.resample_poly` polyphase filtering. (fast)
         'linear'
-            `samplerate` linear interpolation. (very fast)
+            `samplerate` linear interpolation. (very fast, but not bandlimited)
         'zero_order_hold'
-            `samplerate` repeat the last value between samples. (very fast)
+            `samplerate` repeat the last value between samples. (very fast, but not bandlimited)
         'sinc_best', 'sinc_medium' or 'sinc_fastest'
-            `samplerate` high-, medium-, and low-quality sinc interpolation.
+            `samplerate` high-, medium-, and low-quality bandlimited sinc interpolation.
```

Comment:
I also added `bandlimited` here for consistency. It is also stated explicitly in the `libsamplerate` doc: http://www.mega-nerd.com/SRC/api_misc.html